tests/Commands/__snapshots__/ComponentClassCommandTest__it_can_change_the_default_namespace__1.php

```diff
@@ -0,0 +1,26 @@
+<?php return '<?php
+namespace Modules\\Blog\\View\\Component\\newDirectory;
+use Illuminate\\View\\Component;
+class Blog extends Component
+{
+    /**
+     * Create a new component instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
```

Comment:
Remove unused function

Reply:
it tests that changing the path of the component also changes its namespace dynamically

Reply:
You could remove the `__construct()` as it would change a thing afaik